framework/src/st_things/things_stack/easy-setup/easysetup_manager.c

```diff
@@ -294,19 +291,17 @@ esm_result_e esm_init_easysetup(int restart_flag, things_server_builder_s *serve
 	es_resource_mask_e resourcem_mask = ES_WIFI_RESOURCE | ES_CLOUD_RESOURCE | ES_DEVCONF_RESOURCE;
 	// Easy Setup Module Initiation
 
-	if (es_init_enrollee(g_is_secured, resourcem_mask, g_callbacks) != ES_OK) {
+	if (es_init_enrollee(g_is_secured, resourcem_mask, g_callbacks) == ES_ERROR) {
 		THINGS_LOG_ERROR(THINGS_ERROR, TAG, "StartEasySetup and onboarding Fail!!");
```

Comment:
is there any possibility to add another kind of ES_ERRORS ?, or another kind of ES_OK cases?
!= ES_OK also looks good and flexible isn't it?

Reply:
OK. i will change